websites/D/Discords.com/presence.ts

```diff
@@ -3,351 +3,508 @@ const presence = new Presence({
+		];
+	} else if (pathname.includes("/sticker-list/tag")) {
+		presenceData.details = `🏷️ Viewing sticker tag: ${
+			document
+				.querySelector("h2.text-center")
+				?.textContent.replace("Sticker List", " ") ?? "N/A"
 		}`;
+		presenceData.state = `📖 Page ${pageNumber}`;
 		presenceData.buttons = [
 			{
-				label: "View Top-100",
-				url: document.location.href,
+				label: "View Sticker Tag",
+				url: href,
+			},
+		];
+	} else if (pathname.includes("/sticker-list/search")) {
+		const stickSearch = document
+			.querySelector("h2.XJYkMdaM.emojisearchcenter")
+			?.textContent?.split("Search:")[1]
+			?.split("Servers")[0];
+
+		if (stickSearch) presenceData.details = `🔍 Searching for: ${stickSearch}`;
+		presenceData.state = `📖 Page ${pageNumber}`;
+		presenceData.buttons = [
+			{
+				label: "View Results",
+				url: href,
+			},
+		];
+	} else if (pathname === "/bio") {
+		presenceData.details = "🙂 Viewing: Bios";
+		presenceData.state = `📖 Page ${pageNumber}`;
+		presenceData.buttons = [
+			{
+				label: "View Bios",
+				url: href,
 			},
 		];
-	} else if (document.location.pathname.includes("/join")) {
-		const serverNameJoin = document.querySelector("h2.mt-4")?.textContent;
-		presenceData.details = `Looking at 🚦 ${
-			serverNameJoin || "N/A"
-		}'s join page`;
-		if (serverNameJoin) {
+
+		if (standardSearch) {
+			presenceData.details = `🔍 Searching for: ${standardSearch}`;
 			presenceData.buttons = [
 				{
-					label: `Join ${serverNameJoin}`,
-					url: document.location.href,
+					label: "View Results",
+					url: href,
 				},
 			];
 		}
-	} else if (document.location.pathname.includes("/upvote")) {
-		const serverNameUpvote = document.querySelector(
-			"h1.JoinUpvote_clickable-server-name__3l6Es"
+	} else if (pathname.includes("/bio/p")) {
+		presenceData.details = document.querySelector("h1.HiVipESs")?.textContent;
+		presenceData.state = document.querySelector(
+			"div.fHOTdnFx.g0pusrry"
 		)?.textContent;
-		presenceData.details = `Upvoting 🗳️ ${serverNameUpvote || "N/A"} `;
-		if (serverNameUpvote) {
+		presenceData.largeImageKey =
+			document.querySelector("div.jYifcv4i")?.querySelectorAll("img")[1]?.src ??
+			Assets.Logo;
+		presenceData.smallImageKey = Assets.Heart;
+		presenceData.smallImageText =
+			document.querySelector("#like-button")?.textContent === "0"
+				? "Not Liked"
+				: document.querySelector("#like-button")?.textContent;
+		presenceData.buttons = [
+			{
+				label: "View Profile",
+				url: href,
+			},
+		];
+	} else if (pathname.includes("/bots/verification")) {
+		presenceData.details = "🤖 Viewing: Verification";
+		presenceData.buttons = [
+			{
+				label: "View Page",
+				url: href,
+			},
+		];
+
+		if (pathname.includes("/apply")) {
+			presenceData.details = "🤖 Applying for Verification";
 			presenceData.buttons = [
 				{
-					label: `Upvote ${serverNameUpvote}`,
-					url: document.location.href,
+					label: "Apply for Verification",
+					url: href,
 				},
 			];
```

Comment:
Kinda seen as advertisement so wouldn't be permitted

Reply:
Will remove.